Assets/MixedRealityToolkit-SDK/Features/Boundary/System/MixedRealityBoundarySystem.cs

```diff
@@ -182,12 +179,26 @@ public override void Destroy()
             }
         }
 
+        /// <summary>
+        /// Creates the parent for boundary visualization objects that need to teleport with the user.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="GameObject"/> to which boundary visualizations will be parented.
+        /// </returns>
+        private GameObject CreateBoundaryVisualizationParent()
+        {
+            GameObject visualizationParent = new GameObject("Boundary System");
```

Comment:
`Boundary System Visualization`?

I think leaving off the last part is a bit misleading, and might have devs looking for a boundary system component on it, when most likely none will exist.

Reply:
makes sense... i'll update it